telco-examples/mgmt-cluster/single-node/eib/kubernetes/helm/values/neuvector.yaml

```diff
@@ -12,3 +12,6 @@ k3s:
   enabled: true
 crdwebhook:
   enabled: false
+global:
```

Comment:
We also miss the `registry` field here if I'm not mistaken.

Reply:
good catch!!! Copy & paste